### PR TITLE
KeePassXC: Enable and fix build on x86_64

### PR DESCRIPTION
**Pull request by moochris**

### CODE REVIEW

**Comment by diversys** [comment]
You should bump the revision

**Comment by moochris** [comment]
@diversys I tried - 2.2.0 does not build. It fails horribly with namespace problems in the QtConcurrent module.

[ 51%] Building CXX object src/CMakeFiles/keepassx_core.dir/gui/DatabaseOpenWidget.cpp.o
In file included from /boot/system/develop/headers/QtCore/QtConcurrentRun:1:0,
from /sources/keepassxc-2.2.0/src/gui/ChangeMasterKeyWidget.cpp:33:
/boot/system/develop/headers/QtCore/qtconcurrentrun.h:105:61: error: 'HasResultType' is not a member of 'QtPrivate'
auto run(Functor functor) -> typename QtPrivate::QEnableIf<!QtPrivate::HasResultType::Value, QFuture<decltype(functor())> >::Type

**Comment by moochris** [comment]
@diversys Nevermind, you actually meant the revision number in the recipe :) Just doing that...

File: app-admin/keepassxc/patches/keepassxc-2.1.0.patchset [hunk 1]
```diff
@@ -39,5 +39,75 @@ index a77f80a..a688f0a 100644
...
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 916a27c..5135376 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -29,7 +29,7 @@ include(CheckCCompilerFlag)
+ include(CheckCXXCompilerFlag)
+ include(CheckCXXSourceCompiles)
+ 
+-option(WITH_TESTS "Enable building of unit tests" ON)
++option(WITH_TESTS "Enable building of unit tests" OFF)
```
Comment (KapiX) [inline app-admin/keepassxc/patches/keepassxc-2.1.0.patchset]: This is wrong. Disabling unit tests can and should be done in the recipe itself.
See line 77: `-DWITH_TESTS=on`.

Comment (moochris) [inline app-admin/keepassxc/patches/keepassxc-2.1.0.patchset]: Ah, no problem - fixing

**Pull request merged**